packages/web-runtime/src/App.vue

```diff
@@ -83,7 +84,7 @@
 </template>
 <script>
 import 'inert-polyfill'
-import { mapGetters, mapState, mapActions } from 'vuex'
+import {mapActions, mapGetters, mapState} from 'vuex'
```

Comment:
```suggestion
import { mapActions, mapGetters, mapState } from 'vuex'
```
... to make the linter happy ;-)